internal/civisibility/integrations/gotesting/coverage/test_coverage_test.go

```diff
@@ -306,7 +306,7 @@ func TestCollectCoverageAfterTestExecution(t *testing.T) {
 	f.WriteString("mode: count\n")
 	f.Close()
 
-	tc.CollectCoverageAfterTestExecution()
+	tc.getCoverageData()
```

Comment:
Shouldn't we handle the returned error here?

Reply:
fixed in [b795023](https://github.com/DataDog/dd-trace-go/pull/3169/commits/b79502375bfa61693dc16406bb07cffca1eea37a)